calcom/cal.com
# feat: Select darkmode in app.cal.com

## What does this PR do?



Add a section in the appearance page to switch dark/light mode for the Main page(not the booking page).

Solved #11997



https://github.com/calcom/cal.com/assets/71842426/6c165617-283e-4aee-9845-8b3e660985d2


## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is a UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



1. Go to `localhost:3000`
2. Go to your setting page's appearance tab
3.  Switch the "App theme", and you will see the color change of the UI. (Cal.com's system default is dark mode)

## Mandatory Tasks


## Checklist



## Note

Currently, for the wording here, it's only in English for now (I've asked a person from Cal.com in the OSS hackathon, and he said using English for now is good). We need to create another PR to support all other languages.

![Screenshot 2023-12-02 at 12 12 24 PM](https://github.com/calcom/cal.com/assets/71842426/a1ea63bf-bc98-4b4e-9121-97113e698351)
